templates/clientClient.gotmpl

```diff
@@ -82,7 +88,30 @@ func (a *Client) {{ pascalize .Name }}(params *{{ pascalize .Name }}Params{{ if
 		opt(op)
 	}
   {{ $length := len .SuccessResponses }}
-  {{ if .SuccessResponse }}result{{else}}_{{ end }}, err := a.transport.Submit(op)
+  timeout := utils.GetTimeout(a.cfg.RetryTimeout)
+  for n := 0; n <= a.cfg.NumRetries; n++ {
+    // Wait a bit if it is not the first request
+    if n != 0 {
+      time.Sleep(timeout)
```

Comment:
I appreciate this is based off the [original retry logic](https://github.com/grafana/grafana-api-golang-client/blob/303710b3b781ad67f845d344fa2e5afe30db7f31/client.go#L88-L128). Do you know why we went with fixed wait time vs exponential backoff? Curious to see how much time we had to play with here with respect to any request timeout.

Reply:
I'm not sure why it was fixed, however, I added exponential backoff in case the `RetryTimeout` field is set to zero: https://github.com/grafana/grafana-openapi-client-go/pull/11/files#diff-aa33e0acf009f994f053a9386714490fca5ba1509aeba763287ce898f2550106R34-R36 - let me know what you think